spec/cucumber/rb_support/rb_transform_spec.rb

```diff
@@ -9,6 +9,26 @@ def transform(regexp)
       end
 
       describe "#to_s" do
+        it "do not touch positive lookahead captures" do
```

Comment:
Sorry to be petty, but could you change these to read `it "does not touch..."` - that would be better English! :smile: